src/stavisky_lab_to_nwb/stavisky/staviskydecodinginterface.py

```diff
@@ -0,0 +1,166 @@
+"""Primary classes for converting Stavisky decoding data."""
+import redis
+import numpy as np
+from pynwb import NWBFile, TimeSeries
+from ndx_events import LabeledEvents
+from typing import Optional
+
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.tools.nwb_helpers import get_module
+
+
+class StaviskyPhonemeLogitsInterface(BaseDataInterface):
+    """Decoding phoneme logits interface for Stavisky Redis conversion"""
+
+    def __init__(
+        self,
+        port: int,
+        host: str,
+    ):
+        super().__init__(port=port, host=host)
+
+    def add_to_nwbfile(
+        self,
+        nwbfile: NWBFile,
+        metadata: dict,
+        smooth_timestamps: bool = False,  # TODO: removal/modification pending timestamp meeting
+    ):
+        # initialize redis client and check connection
+        r = redis.Redis(
+            port=self.source_data["port"],
+            host=self.source_data["host"],
+        )
+        r.ping()
+
+        # get processing module
+        module_name = "decoding"
+        module_description = "Contains decoder outputs for real-time speech decoding"
```

Comment:
```suggestion
        module_description = "Contains decoder outputs for real-time speech decoding."
```

Description is fine even if you change to `"behavior"` - I do prefer though if they form complete sentences (i.e., punctuation, even if only a single sentence)